README.md

```diff
@@ -1,12 +1,18 @@
 The 18F Method Cards (“18F Methods”) describe how 18F puts human-centered design into practice. While this is primarily maintained as an internal resource, we hope it can help *everyone* adopt the methods of human-centered design.
 
-### Why methods?
+## 18F Methods mission and vision
+In late 2016, we articulated a vision and mission for the project that focus our efforts on meeting the needs of government users:
```

Comment:
I would suggest removing the "In late 2016... sentence"

Reply:
Agreed.

Also might want to change "*everyone*" to "*anyone*"